geoips/plugins/modules/procflows/config_based.py

```diff
@@ -1392,7 +1394,9 @@ def config_based(fnames, command_line_args=None):
                         output_dict=output_dict,
                         covg_args_field_name="image_production_covg_args",
                     )
-                    covg = covg_func(alg_xarray, product_name, area_def, **covg_args)
+                    covg = covg_func.call(
```

Comment:
@jsolbrig Updated in single_source, config_based, and title_formatters to add "call" to the entry point (since now entry points point to module vs function)

Reply:
They should just be able to be called without this. If they're not working, something is wrong. They should all have `__call__` methods now.

Reply:
After talking, I understand why this was done. This is fine for now. Will want to remove later.